zuul.d/projects.yaml

```diff
@@ -16,3 +16,37 @@
         - edpm-ansible-molecule-edpm_ovn_bgp_agent
         - edpm-ansible-molecule-edpm_ovs
         - edpm-ansible-molecule-edpm_tripleo_cleanup
+        - openstack-k8s-operators-content-provider
+        - cifmw-data-plane-adoption-osp-17-to-extracted-crc:
+            dependencies:
+              - openstack-k8s-operators-content-provider
+            # files based on https://github.com/marios/data-plane-adoption/blob/ec0f395a8321dc4c3ced71b777102ad7dc9de1b2/tests/roles/dataplane_adoption/tasks/main.yaml#L256
```

Comment:
i updated the files: list based on yesterday's conversation.

As far as I can see these are the specific plays and roles we are consuming in this job cc @slagle @jpodivin @jistr @rebtoor @cescgina

Reply:
ack fair enough I will update and remove plugins 

and I'll add the pre-adoption-validation

Reply:
Awesome.